README.md

```diff
@@ -84,6 +84,32 @@ Creating a CSV export for Carbonite Migration
     1. Enter the location of the CSV file generated on D42 website. The source machine IP will be parsed from this file.
     2. The "Carbonite target IP" is the replication server and the server added to the Carbonite console application. This machine stores the protected data.
     3. The "ESX host IP/HyperV host IP" is the ESXi/HyperV VM Server which will host the replica of your source machine.
+    4. When asked for the type of job you will be creating, please be aware that the available choices differ for Windows and Linux OSs. The following jobs are available: 
+
+        **Availability for Windows jobs:**
+        - **FilesAndFolders** — Files and folders
+        - **ClusterAwareFilesAndFolders** — Cluster-aware files and folders
+        - **SQL** — SQL
+        - **ClusterAwareSql** — Cluster-aware SQL
+        - **FullServerFailover** — Full server
+        - **VRA** — Full server to ESX or full server to Hyper-V
+
+        **Migrate for Windows jobs:**
+        - **MoveDataOnlyMigration** — Files and folders migration
+        - **MoveServerMigration** — Full server migration
+        - **VraMove** — Full server to ESX migration or full server to Hyper-V migration
+
+        **Availability for Linux jobs:**    
+        - **LinuxFilesAndFolders** — Files and folders
+        - **LinuxFullServerFailover** — Full server
+        - **Lvra** — Full server to ESX
+
+        **Migrate for Linux jobs:**
+        - **LinuxMoveServerMigration** — Full server migration
+        - **MoveLvra** — Full server to ESX migration
```

Comment:
Can you do a Hyper-V migration for Linux?

Reply:
Sure. I will try this with the existing HyperV machine.